src/index.html

```diff
@@ -5,8 +5,342 @@
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a href="https://www.moyo.ua/ua/" class="header__logo">
+        <img src="images\logo.png" alt="logo">
+      </a>
+      <nav>
+        <li class="header__list">
+          <ul class="header__list-item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple/"
+              class="header__link is-active"
+            >
+              Apple
+            </a>
+          </ul>
+
+          <ul class="header__list-item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple/"
+              class="header__link"
+            >
+              Samsung
+            </a>
+          </ul>
+
+          <ul class="header__list-item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple/"
+              class="header__link"
+            >
+              Smartphones
+            </a>
+          </ul>
+
+          <ul class="header__list-item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple/"
+              class="header__link"
+              data-qa="nav-hover"
+            >
+              Laptops&nbsp&&nbspcomputers
+            </a>
+          </ul>
+
+          <ul class="header__list-item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple/"
+              class="header__link"
+            >
+              Gadgets
+            </a>
+          </ul>
+
+          <ul class="header__list-item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple/"
+              class="header__link"
+            >
+              Tablets
+            </a>
+          </ul>
+
+          <ul class="header__list-item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple/"
+              class="header__link"
+            >
+              Photo
+            </a>
+          </ul>
+
+          <ul class="header__list-item">
+            <a
+              href="https://www.moyo.ua/ua/portal_brand/apple/"
+              class="header__link"
+            >
+              Video
+            </a>
+          </ul>
+        </li>
+      </nav>
+    </header>
+
+    <main class="container">
+      <article class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          class="card__image"
+          alt="APPLE A1419 iMac 27"
+        >
+        <h2 class="h2 card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+        <p class="p card__product-code">Product code: 195434</p>
+        <div class="card__rating">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
```

Comment:
don`t forget to add empty line between sibling tags

Reply:
added